doc/en/source/api.rst

```diff
@@ -1143,6 +1143,977 @@ Here is the Javascript-based code example to terminate a service topology by del
     });    
 
 
+NGSI-LD Supported API's
+============================
 
+The following figure shows a brief overview of how the APIs in current scope will be used to achieve the goal of NGSI-LD API support in FogFlow. The API support includes Entity creation, registration, subscription and notification.
 
 
+
+.. figure:: figures/ngsild_architecture.png
+
+
+
+
+Entities API
+------------
+For the purpose of interaction with Fogflow, IOT devices approaches broker with entity creation request where it is expanded as per given context. Broker further forwards the registration request to Fogflow Discovery in correspondence to the created entity.
```

Comment:
[minor]better to write understood or resolved as per given context.

Reply:
done